src/color.coffee

```diff
@@ -0,0 +1,88 @@
+_ = require 'underscore-plus'
+ParsedColor = require 'color'
+
+# Essential: A simple Color class returned from `atom.config.get` when the value
+# at the key path is of type 'color'.
+module.exports =
+class Color
+  # Essential: Parse a {String} or {Object} into a {Color}.
+  #
+  # * `value` - A {String} such as `'white'`, `#ff00ff`, or
+  #             `'rgba(255, 15, 60, .75)'` or an {Object} with `red`, `green`,
+  #             `blue`, and `alpha` properties.
+  #
+  # Returns a {Color} or `null` if it cannot be parsed.
+  @parse: (value) ->
+    return null if _.isArray(value)
+    return null if _.isFunction(value)
+    return null unless _.isObject(value) or _.isString(value)
+
+    try
+      parsedColor = new ParsedColor(value)
+    catch error
+      return null
+
+    new Color(parsedColor.red(), parsedColor.green(), parsedColor.blue(), parsedColor.alpha())
+
+  constructor: (red, green, blue, alpha) ->
+    Object.defineProperties this,
```

Comment:
This allows `JSON.stringify` to still work on `Color` objects while enforcing that color values are always valid.